cmd/lakefs/cmd/root.go

```diff
@@ -51,6 +51,13 @@ func loadConfig() *config.Config {
 		fmt.Println("Failed to load config file", err)
 		os.Exit(1)
 	}
+	logger := logging.Default().WithField("phase", "startup")
+	prettyCfg, err := cfg.GetConfigAsJson()
+	if err != nil {
+		fmt.Println("Failed getting configuration JSON", err)
+		os.Exit(1)
+	}
+	logger.Trace("Configuration loaded:", string(prettyCfg))
```

Comment:
Why do we need this one?

Reply:
Since configuration is changed dynamically - I think it will be beneficial to get the current configuration to log whenever we load it